src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt

```diff
@@ -164,6 +164,8 @@ const val PASSWORD_REGEX_HINT = "Password should contain at least one uppercase
     "one lowercase letter, one number and one special character[@ _ #]"
 const val ORGANIZATION_ID_REGEX = "^[a-zA-Z0-9]*\$"
 const val ORGANIZATION_ID_REGEX_HINT = "Organization ID should be a valid alphanumeric string"
+const val ORGANIZATION_NAME_REGEX = "^[_a-zA-Z0-9-\\s]*\$"
```

Comment:
Is there any reason for `organization name` to be different from `resource name`? If not, can we have the same regex (with space) for orgName as well as resourceName?

Reply:
Resource names are stored as part of hrn,
So if we allow resource name to contain [space], then our hrn will also contain spaces like
`hrn:orgId::resource name`
Is it fine to have space in hrn?